docs/rules/id-match.md

```diff
@@ -0,0 +1,72 @@
+# Require IdMatch (id-match)
+
+"There are only two hard things in Computer Science: cache invalidation and naming things." � Phil Karlton
```

Comment:
Inserting a ">" at the beginning of this line and a blank line after will format it as a blockquote if that's what you're going for.